README.md

```diff
@@ -1,30 +1,70 @@
-# Unirep monorepo
+<p align="center">
+    <h1 align="center">Unirep Protocol</h1>
+</p>
 
-## About Unirep
+<p align="center">
+    <a href="https://github.com/unirep/unirep">
+        <img src="https://img.shields.io/badge/project-unirep-blue.svg?style=flat-square">
+    </a>
+    <a href="https://github.com/unirep/unirep/blob/master/LICENSE">
+        <img alt="Github license" src="https://img.shields.io/github/license/unirep/unirep.svg?style=flat-square">
+    </a>
+    <a href="https://eslint.org/">
+        <img alt="Linter eslint" src="https://img.shields.io/badge/linter-eslint-8080f2?style=flat-square&logo=eslint">
+    </a>
+    <a href="https://prettier.io/">
+        <img alt="Code style prettier" src="https://img.shields.io/badge/code%20style-prettier-f8bc45?style=flat-square&logo=prettier">
+    </a>
+    <a href="https://dl.circleci.com/status-badge/redirect/gh/Unirep/Unirep/tree/develop">
+        <img alt="Circle CI" src="https://dl.circleci.com/status-badge/img/gh/Unirep/Unirep/tree/develop.svg?style=svg">
```

Comment:
Build badge needs to be updated. Consider using the flat one from shields to match the others:

![CircleCI](https://img.shields.io/circleci/build/github/Unirep/Unirep/main?style=flat-square)